src/renderer/window/input_state.cpp

```
@@ -45,10 +47,24 @@ mr::InputState::KeyCallbackT mr::InputState::get_key_callback() noexcept
   return [this](const vkfw::Window &window, vkfw::Key key, int scan_code,
                 vkfw::KeyAction action, vkfw::ModifierKeyFlags flags) {
     std::lock_guard lock(update_mutex);
+    auto idx = std::to_underlying(key);
+
+    if (idx < 0 && idx >= _writer_key_tapped.size()) {
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)